### ISSUE TITLE
Invalid song dates make Rofi_MPD crash

### ISSUE THREAD

**Issue by monsieurh** [JakeStanger/Rofi_MPD#4]
When I try to run the program I have the following error:
```

Traceback (most recent call last):
  File "main.py", line 119, in <module>
    'epoch': get_album_release_epoch(song_data=song),
  File "main.py", line 86, in get_album_release_epoch
    epoch = datetime.datetime(int(year), int(month), int(day))
ValueError: month must be in 1..12
```

I assume some of my songs have invalid dates

**Comment by rubdos** [comment]
related #3

**Comment by JakeStanger** [comment]
Fixed by #5 for now

**Comment by rubdos** [comment]
Another one:

```
Traceback (most recent call last):
  File "/usr/bin/rofi-mpd", line 122, in <module>
    'epoch': get_album_release_epoch(song_data=song),
  File "/usr/bin/rofi-mpd", line 78, in get_album_release_epoch
    epoch = datetime.datetime(year, 1, 1)
ValueError: year 0 is out of range
```

I indeed have an album with year 0 :-)

**Comment by JakeStanger** [comment]
Addressed in #3
